docs/Plugins/webhook.md

```diff
@@ -100,27 +94,39 @@ Read more in Swagger: http://localhost:4000/api/swagger/index.html#/plugins%2Fwe
 
 ## Deployments
 
-Adding pipeline webhook in your deploy shells will help you collect data into DevLake.
+If your CI/CD tool isn't already supported by DevLake, you can insert curl commands in your CI/CD script to post deployment data to DevLake.
+
+DevLake models CI/CD domain with two entities: Pipelines and Tasks. A pipeline has many tasks.
+
+#### Example 1
 
-First, let us know there are two entities: Tasks and Pipelines. A pipeline means one build or deployment, and a pipeline may have more than one task.
+Take GitHub Actions as an example, the screenshot below shows a GitHub Actions workflow, which consists of 6 jobs including `golangci-lint`, `unit-test`, and etc. When DevLake normalizes data from GitHub Actions, it creates the following records in the domain layer:
+
+- 1 entry in cicd_pipelines table, corresponds to the GitHub workflow
+- 6 entries in cicd_tasks table, one for each job in the GitHub workflow
 
 ![image](https://user-images.githubusercontent.com/3294100/191319143-ea5e9546-1c6d-4b2a-abba-95375cfdcec3.png)
 
-(Example 1)
+#### Example 2
+
+Then look at Jenkins build history, the screenshot below shows 12 Jenkins builds, and no Jenkins stages appeared. When DevLake normalizes data from Jenkins build history, it creates the following records in the domain layer:
```

Comment:
Now look at the Jenkins build history, as another example: the screenshot below shows 12 Jenkins builds, with no Jenkins stages: